### ISSUE TITLE
After import collection remains empty

### ISSUE THREAD

**Issue by denis-gol** [getinsomnia/insomnia#5045]
### Expected Behavior

after import, the collection becomes full, the block with the environment is also filled

### Actual Behavior

nothing is added when importing, the collection remains empty

### Reproduction Steps

1. export collection (json|yaml) 
2. import file in a new collection

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

![image](https://user-images.githubusercontent.com/38101819/182842387-4c948a7d-a118-4bc0-8543-ce0a7c9bda3a.png)
![image](https://user-images.githubusercontent.com/38101819/182842411-35103718-560d-41e9-8225-c539bca06b58.png)

### Insomnia Version

2022.5.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Corporate LTSC

### Installation method

download from insomnia

**Comment by filfreire** [comment]
Hi @denis-gol, thanks for reporting this!

This issue might be a duplicate of https://github.com/Kong/insomnia/issues/4274, in any case I would recommend the following:

- Double check if your imported data is by any chance going to another `New Document` you might have somewhere on your dashboard (similar to [this comment](https://github.com/Kong/insomnia/issues/4274#issuecomment-1102753929))

- Double check if when you're importing the collection, if it's not by mistake getting imported into another Insomnia Project or Team space.
![Pasted_Image_05_08_2022__09_20](https://user-images.githubusercontent.com/11976836/183034817-5249345b-4b08-4c53-ba9f-8f8881d56e48.png)

Let us know if any of these helped, and feel free to report any other issues you come across.

**Comment by copiri-six** [comment]
👍  I have this issue as well.  In my case, it did in fact turn out that the collection was being imported into another new document.  Meaning, the following steps occurred:

- Open Insomnia
- Create New Document
- Go to Debug, import from file (note: why doesn't the import from file in the Design tab work for this?!?)
- Import succeeds as shown above, but nothing changes
- Go back out to the list of collections, notice that a new one has been added

For me, this bug is actually "why isn't the import being done where I expect it to be, which is in the Document that I clicked the 'Import from File' button in?"

As others have noted in the linked issue, part of the problem is that sometimes the import overwrites something else.  I check/test collections from others, and I don't understand why an updated import from them can't be kept separately in the new Document that I just created (you know... where I clicked the import button...), regardless of its collection ID.

I am sure there are lots of reasons for a collection ID to be global, but it seems to me that there should be a warning of some kind before Insomnia decides that not only is it going to ignore my expected placement of the import, but it's going to overwrite something I already have.
